ci/env.sh

```diff
@@ -0,0 +1,21 @@
+#smoke test environment specifications:
+
+#for scale perf
+export SCALE=4
+export RUNS=1
+
+#for kube burner 
+export JOB_ITERATIONS=3
+export CLEANUP=true
+export CLEANUP_WHEN_FINISH=true
+export NODE_COUNT=4
```

Comment:
This depends on the number of the nodes running in the cluster on which the test is being run, maybe set this is to the node count present in the cluster being used by benchmark-wrapper? Thoughts?

Reply:
yes we could do that